packages/google-cloud-run/esm-loader.mjs

```diff
@@ -24,3 +24,6 @@
  */
 
 import './src/index.js';
+// Here we exports all named exports from '@instana/core/iitm-loader.mjs', enabling 
+// integration of import-in-the-middle (IITM) for Native ESM module support.
+export * from '@instana/core/iitm-loader.mjs';
```

Comment:
question: did you do a performance comparison?
- App uses collector v3.10.0 & ESM
 - Updates to this version (now iitm gets also required)
 - compare the loading times (bootstrap times)

Reply:
I will do the performance testing asap and will pin the results here

Reply:
The performance test results for the Instana Collector with different configurations:

| Test Type             | Measure Method                    |Avg. Bootstrap Time   |
|-----------------------|---------------------------------|---------------------|
| Without iitmhook      | console.time                     | 100ms to 200ms         |
|  With iitmhook | console.time             | 100ms to 200ms         |

No significant performance variation was observed in CI tests [with](https://cloud.ibm.com/devops/pipelines/tekton/c2cd6a8d-ea5a-47b0-913e-cd172d63833f/runs/4bb0bde1-d8da-4aa0-8467-d567b43b9c1d?env_id=ibm:yp:eu-de) and [without](https://cloud.ibm.com/devops/pipelines/tekton/c2cd6a8d-ea5a-47b0-913e-cd172d63833f/runs/4bb0bde1-d8da-4aa0-8467-d567b43b9c1d/echo-env/execute?env_id=ibm:yp:eu-de) iitmhook for esm apps.